src/components/atoms/file-upload-field/index.tsx

```diff
@@ -5,16 +5,25 @@ type FileUploadFieldProps = {
   onFileChosen: (files: any[]) => void
   filetypes: string[]
   errorMessage?: string
-  placeholder?: string
+  placeholder?: React.ReactElement | string
   className?: string
+  text?: React.ReactElement | string
 }
 
+const defaultText = (
+  <span>
```

Comment:
question: does {’ ‘} is equivalent to &nbsp; ? Because i think this one can be break and the other one is unbreakable and I tend to use the second one